src/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sabka Bazaar</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Dosis:400,600,700">
+</head>
+
+<body>
+    <header id="header"></header>
+    <section class="slideshow-section">
+        <div class="row" style="display: inline;">
+            <div class="slideshow-container" id="slideshow">
+                <a class="prev" onclick="plusSlides(-1)">PREV</a>
+                <a class="next" id="next" onclick="plusSlides(1)">NEXT</a>
+            </div>
+            <br>
+
+            <div style="text-align:center" id="slideshow-row__dot"></div>
+            <script type="text/x-handlebars-template" id="slideshow-row__dotTemplates">
```

Comment:
Handlebar templates can be created as separate JS files and required in the mail js file

Reply:
DONE